torch/csrc/jit/codegen/cuda/fusion.h

```diff
@@ -209,6 +213,20 @@ struct TORCH_CUDA_API Fusion : public IRInputOutput {
   bool hasGridReduction();
   size_t gridReductionTempBufferSize();
 
+  void setValuesMap(std::unordered_map<Val*, Val*> values_map) {
+    values_map_ = std::move(values_map);
+  }
+
+  Val* loweredVal(Val* value) const {
+    auto it = values_map_.find(value);
+    return it != values_map_.end() ? it->second : value;
```

Comment:
What's the significance if we look for a value in the lowered map and don't find one?

Reply:
Sorry, this became more clear when I got later in the PR, didn't review my review before sending.